release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -614,26 +614,6 @@ See
 xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating
 Container Logs] for more information.
```

Comment:
@anpingli Please disregard the changes in this file. This is a manual cherrypick of https://github.com/openshift/openshift-docs/pull/12234. I committed this change here so that I would not forget to make the change later. Please review the changes in https://github.com/openshift/openshift-docs/pull/12572/files#diff-40f3b73686ab98d0e3b12958011ae7bf